src/components/NewSectionSegmentPopup.jsx

```diff
@@ -0,0 +1,112 @@
+import { useState, React } from 'react';
+import {
+  Button,
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  RadioGroup,
+  Radio,
+  Stack,
+  Input,
+  Text,
+} from '@chakra-ui/react';
+import { AddIcon } from '@chakra-ui/icons';
+import Section from './Section';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Section' is defined but never used.